#!/usr/bin/env python
PACKAGE = "phx_workstation_override"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("motor0", int_t, 0, "Front", 2000, 1700, 3500)
gen.add("motor1", int_t, 0, "Right", 2000, 1700, 3500)
gen.add("motor2", int_t, 0, "Back", 2000, 1700, 3500)
gen.add("motor3", int_t, 0, "Left", 2000, 1700, 3500)


exit(gen.generate(PACKAGE, "phx_workstation_override", "WorkstationOverride"))
